README.md

```diff
@@ -125,31 +145,55 @@ To prevent the extension from adding the validation badges next to the resources
 
 ### Data Validation
 
-CKAN users will be familiar with the validation performed against the metadata fields when creating or updating datasets. The form will return an error, for instance, if a field is missing or it doesn't have the expected format.
-
-Data validation follows the same principle, but against the actual data published in CKAN, that is the contents of tabular files (Excel, CSV, etc) hosted in CKAN itself or elsewhere. Whenever a resource of the appropriate format is created or updated, the extension will validate the data against a collection of checks. This validation is powered by [Goodtables](https://github.com/frictionlessdata/goodtables-py), a very powerful data validation library developed by [Open Knowledge International](https://okfn.org) as part of the [Frictionless Data](https://frictionlessdata.io) project. Goodtables provides an extensive suite of [checks](https://github.com/frictionlessdata/goodtables-py#checks) that cover common issues with tabular data files.
-
-These checks include structural problems like missing headers or values, blank rows, etc., but also can validate the data contents themselves (see [Data Schemas](#data-schemas)) or even run [custom checks](https://github.com/frictionlessdata/goodtables-py#custom-constraint).
-
-The result of this validation is a JSON report. This report contains all the issues found (if any) with their relevant context (row number, columns involved, etc). The reports are stored in the database and linked to the CKAN resources, and can be retrieved [via the API](#resource_validation_show).
-
-If there is a report available for a particular resource, a status badge will be displayed in the resource listing and on the resource page, showing whether validation passed or failed for the resource.
+CKAN users will be familiar with the validation performed against the metadata
+fields when creating or updating datasets. The form will return an error, for
+instance, if a field is missing or it doesn't have the expected format.
+
+Data validation follows the same principle, but against the actual data
+published in CKAN, that is the contents of tabular files (Excel, CSV, etc)
+hosted in CKAN itself or elsewhere. Whenever a resource of the appropriate
+format is created or updated, the extension will validate the data against a
+collection of checks. This validation is powered by
+[Goodtables](https://github.com/frictionlessdata/goodtables-py), a very
+powerful data validation library developed by [Open Knowledge International](https://okfn.org) 
+as part of the [Frictionless Data](https://frictionlessdata.io) project.
+Goodtables provides an extensive suite of [checks](https://github.com/frictionlessdata/goodtables-py#checks)
```

Comment:
```suggestion
Frictionless Framework provides an extensive suite of [checks](https://framework.frictionlessdata.io/docs/checks/baseline.html)
```
Please update if there's a better link. It would be better to link to the parent "Validation checks" header but I don't think that's possible

Reply:
Yes, it's not possible to link to the top "Validation checks", I think this is the best link for now.